ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/BlockFetch/Server.hs

```diff
@@ -107,17 +108,18 @@ blockFetchServer _tracer chainDB _version registry = senderSide
         -- When we got an iterator, it will stream at least one block since
         -- its bounds are inclusive, so we don't have to check whether the
         -- iterator is empty.
-        Right it -> SendMsgStartBatch $ sendBlocks it
+        Right it -> SendMsgStartBatch $ sendBlocks 0 it
 
-    sendBlocks :: ChainDB.Iterator m blk (WithPoint blk (Serialised blk))
+    sendBlocks :: Int -> ChainDB.Iterator m blk (WithPoint blk (Serialised blk))
                -> m (BlockFetchSendBlocks (Serialised blk) (Point blk) m ())
-    sendBlocks it = do
+    sendBlocks !blockCount it = do
```

Comment:
Please remove this count; the downstream tracer in `cardano-node` is doing the counting there.

Reply:
Removed.  Thanks!